docusaurus.config.js

```diff
@@ -1,53 +1,55 @@
 // @ts-check
 // Note: type annotations allow type checking and IDEs autocompletion
 
-const lightCodeTheme = require('prism-react-renderer/themes/github');
-const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+const lightCodeTheme = require("prism-react-renderer/themes/github");
+const darkCodeTheme = require("prism-react-renderer/themes/dracula");
 
 /** @type {import('@docusaurus/types').Config} */
 const config = {
-  title: 'Painite',
-  tagline: 'Painite project - to-go website for nitt students for all academic infomations.',
-  url: 'https://painite.nitt.edu',
-  baseUrl: '/',
-  onBrokenLinks: 'throw',
-  onBrokenMarkdownLinks: 'warn',
-  favicon: 'img/favicon.ico',
+  title: "Painite",
+  tagline:
+    "Painite project - to-go website for nitt students for all academic infomations.",
+  // url: "https://painite.nitt.edu",
+  url: "http://192.168.1.3:3000/",
```

Comment:
This url is kept when we deploy the website. Are you sure we should keep the local URL as website. Let me know what you think.

Reply:
I kept the local URL for the time being. We need to change it once we deploy.